packages/desktop/src/index.js

```diff
@@ -356,6 +357,23 @@ class Interpreter {
           event.preventDefault();
         }
 
+        if (event.type == "submit") {
+          event.preventDefault();
+        }
+        
+        if (event.type == "click") {
+          event.preventDefault();
+          if (should_prevent_default !== `onclick`) {
+            console.log(event.target.getAttribute("href"));
```

Comment:
Just remove this console log and then everything looks pretty good.

I would also make a check that "href" exists on the `<a>` tag so we don't send null and panic the Rust code.